python/lsst/ts/dimm/controllers/soar_dimm.py

```diff
@@ -0,0 +1,99 @@
+
+import asyncio
+import sqlalchemy
+import warnings
+
+from .base_dimm import BaseDIMM, DIMMStatus
+
+
+__all__ = ['SOARDIMM']
+
+
+class SOARDIMM(BaseDIMM):
+    """This controller provides an interface with the SOAR telescope DIMM. This will connect to their sql database
+    and publish the data to LSST middleware.
+
+    This controller class is still under development.
+    """
+
+    def __init__(self):
+        super().__init__()
+
+        warnings.warn("This class is still under development and will not work as expected.")
```

Comment:
What's not working here? It might be good to list that out. I don't know thats its super important.

Reply:
yeah... I'll add more info here...